.travis.yml

```diff
@@ -3,11 +3,11 @@ language: objective-c
 xcode_workspace: WordPress.xcworkspace
 xcode_scheme: WordPress
 xcode_sdk: iphonesimulator
+cache:
+- bundler
+- cocoapods
 notifications:
   email: true
-install:
-- gem install xcpretty-travis-formatter
-- pod install
```

Comment:
Travis runs `pod install` by default, as long as there's a `Podfile` and you don't override the `install` step

https://docs.travis-ci.com/user/languages/objective-c#Dependency-Management